multiaccounts/settings/database.go

```diff
@@ -185,6 +186,8 @@ INSERT INTO settings (
 		s.TestNetworksEnabled,
 		s.Fleet,
 		s.AutoRefreshTokensEnabled,
+		// Default the news feed last fetched timestamp to now
+		time.Now().Unix(),
```

Comment:
Is the intention of this line to assign the timestamp value (or a default) to the `s.NewsFeedLastFetchedTimestamp` field? If so, I'm not sure I follow how this works, it seems like there's no connection to assigning the field. Thoughts?